src/main/scala/nl.knaw.dans.easy.deposit/StateManager.scala

```diff
@@ -115,31 +119,57 @@ case class StateManager(depositDir: File, submitBase: File, easyHome: URL) exten
     UUID.fromString(draftProps.getString(bagIdKey))
   }
 
-  private def landingPage = {
-    Try { getProp("identifier.fedora", submittedProps) }
+  private def mailToDansMessage: String = {
+
+    val title: String = (draftDeposit.getDatasetMetadata.map(_.titles) match {
+      case Success(Some(titles: Seq[String])) => titles.headOption
+      case _ => None
+    }).getOrElse("").trim()
+    val shortTitle = title.substring(0, Math.min(42, title.length)).replaceAll("[\r\n]+", " ")
+    val mediumTitle = title.substring(0, Math.min(1000, title.length))
+    val ref = getSubmittedBagId.map(_.toString).getOrElse(s"DRAFT/$relativeDraftDir")
+    val subject = queryPartEncode(s"${ shortTitle } (reference nr: $ref)")
+    val body = queryPartEncode(
+      s"""Hello
+         |
+         |Could you please figure out what went wrong with my deposit?
+         |
+         |It has reference:
+         |   $ref
+         |and title:
+         |   $mediumTitle
+         |""".stripMargin
```

Comment:
```suggestion
         |
         |Kind regards,
         |$userFullName""".stripMargin
```

Reply:
the `userFullName` is not yet defined; I'm not sure if it is easily doable to fetch the user's full name here. Otherwise maybe the username will do?

Reply:
I'm afraid only ldap can provide the full user name. Indeed quite elaborate and probably error-prone.

Reply:
I was afraid you would say that indeed 😉 
Perhaps the username would suffice?